### ISSUE TITLE
Create SPDX-2.2 JSON instead of current SPDX-2.1 tag/value

### ISSUE THREAD

**Issue by maxhbr** [fsfe/reuse-tool#411]
The current implementation generates the hard to parse and less supported SPDX-2.1 document in the tag/value format. It would be a huge step in the direction of simplifying integration and tool support, if the output would be SPDX-2.2 JSON or YAML.

**Comment by maxhbr** [comment]
see also: https://github.com/fsfe/reuse-tool/issues/394

**Comment by mxmehl** [comment]
Absolutely, but I am afraid this issue duplicated #394 as well as #183 (and other JSON/YAML related issues). So I'll close it but amend the former issue accordingly.
